azure_monitor/src/azure_monitor/sdk/auto_collection/live_metrics/__init__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#
+import time
+import typing
+import uuid
+
+from azure_monitor.protocol import BaseObject, LiveMetricEnvelope
+from azure_monitor.utils import azure_monitor_context
+
+LIVE_METRICS_SUBSCRIBED_HEADER = "x-ms-qps-subscribed"
+STREAM_ID = str(uuid.uuid4())
```

Comment:
These should be unique to each manager instance that is created, not on import of this file.

Reply:
Spec never mention what you are suggesting or I could find that part, are you sure about this?, global ID by Application is current behavior in node.js SDK as well

Reply:
Oh if the ID represents the application rather than the stream instance then that's fine.